Classes/Provider/Leaflet.php

```diff
@@ -253,12 +261,12 @@ protected function getMarker($item, $table)
                         $iconOptions['html'] = $marker['icon'];
                         $markerOptions['icon'] = 'icon: new L.divIcon(' . json_encode($iconOptions) . ')';
                     } else {
-                        $icon = $GLOBALS['TSFE']->absRefPrefix . $marker['icon']->getPublicUrl();
+                        $icon = $GLOBALS['TSFE']->absRefPrefix . '/' . $marker['icon']->getPublicUrl();
```

Comment:
This leads to invalid paths like `//fileadmin/ods_osm/marker-60x60.png`

Reply:
Removed it.